DQM/GEM/plugins/GEMRecHitSource.cc

```diff
@@ -184,21 +197,21 @@ void GEMRecHitSource::analyze(edm::Event const& event, edm::EventSetup const& ev
       for (auto hit = gemRecHit; hit != recHitsRange.second; ++hit) {
         GlobalPoint recHitGP = GEMGeometry_->idToDet(hit->gemId())->surface().toGlobal(hit->localPosition());
         Float_t fPhi = recHitGP.phi();
-        if (fPhi < -5.0 / 180.0 * 3.141592)
-          fPhi += 2 * 3.141592;
 
         // Filling of RecHit occupancy
         mapTotalRecHit_layer_.Fill(key3, chamber, eId.ieta());
 
         // Filling of R-Phi occupancy
         Float_t fR = fRadiusMin_ + (fRadiusMax_ - fRadiusMin_) * (eId.ieta() - 0.5) / stationInfo.nNumEtaPartitions_;
-        mapRecHitWheel_layer_.Fill(key3, fPhi, fR);
+        Float_t fPhiShift = (fPhi >= stationInfo.fMinPhi_ ? fPhi : fPhi + 2 * 3.141592);
```

Comment:
Please use M_PI instead of 3.14...

Reply:
Hi @sroychow, 

Thanks for your advice. All 3.141592 have been replaced. Please let me know if there is any missed.

Best regards, 
Byeonghak Ko